app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -156,11 +156,19 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
         let offSetDate = new Date();
         offSetDate.setDate(offSetDate.getDate() - offSet);
         offSetDate.setHours(0, 0, 0);
-        const currStartDate = filter.startDate ? new Date(filter.startDate) : offSetDate;
+        // if no date offset/ start date this means the user is looking for all content
+        // unix epoch used as a default start date for all content
+        const currStartDate = filter.startDate
+          ? new Date(filter.startDate)
+          : filter.dateOffset !== undefined
+          ? offSetDate
+          : new Date(1970, 0, 1);
```

Comment:
Set start date as Unix Epoch for when 'All' is selected (farthest back we can go)